src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -165,13 +164,13 @@ export class SimpleCompletionModel {
 				} else {
 					// by default match `word` against the `label`
 					const match = scoreFn(word, wordLow, wordPos, item.completion.label, item.labelLow, 0, this._fuzzyScoreOptions);
-					if (!match) {
+					if (!match && word !== '') {
```

Comment:
Not sure why this was needed?

Reply:
match is undefined if the prompt is empty. this is part of the fix for https://github.com/microsoft/vscode/issues/233805

Reply:
This worked before in pwsh fine though 🤔

Reply:
Pretty sure it didn't work before. Just showed global completions. I'll check on stable